content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -156,6 +156,18 @@ To ignore multiple codes for a resource, separate each code with a comma:
 $ kubectl annotate deployment my-deployment galley.istio.io/analyze-suppress=IST0107,IST0002
 {{< /text >}}
 
+### Cleanup
```

Comment:
why do we need cleanup section for istioctl analyze?

Reply:
No, I don't think this makes sense. Cleanup instructions are only supposed to remove resources that are created in the task, but none of these things are created by instructions in this doc.

Reply:
I think I was looking at the test files for places where we were deleting Istio installation by just deleting the validating and mutating webhoook configuration and then deleting the istio-system ns. You are right, we should remove all those lines where resources are not created by this task. The page does have a line about installing istio ` istioctl install --set values.global.istiod.enableAnalysis=true`. But I agree it doesn't make sense to uninstall something that the task hasn't specifically required to install.

Reply:
> why do we need cleanup section for istioctl analyze?

We don't want a cleanup section in this doc because it isn't a task that tells the user to do things that need to be cleaned up. To test the snips, the test program sets up a test environment that needs to be cleaned up, but that's outside of the document.